test/uk/gov/hmrc/incometaxsoftwarechoicesfrontend/helpers/MessagesMatcher.scala

```diff
@@ -26,11 +26,13 @@ trait MessagesMatcher {
 
   def allBeIn(messages: Set[String]): MessagesMatcher = MessagesMatcher(messages)
 
-  def containUniqueKeys: DuplicateMatcher = DuplicateMatcher()
+  def containUniqueKeys: Matcher[List[String]] = DuplicateMatcher()
```

Comment:
Why was the type changed from `DuplicateMatcher` to `Matcher[List[String]]`?

Reply:
Because that's what's actually needed by the test class.  Good practice to narrow the type IMO.

I did the same on my additional matcher.

Reply:
Actually, you are broadening the type by using `Matcher[List[String]]`, `DuplicateMatcher` is more narrow.